frontend/src/components/Navigator/components/NavigatorHeader.js

```diff
@@ -5,24 +5,25 @@
 import React, { useEffect } from 'react';
 import PropTypes from 'prop-types';
 
-const NavigatorHeader = ({ label }) => {
+const NavigatorHeader = ({ key, label }) => {
   useEffect(() => {
     document.getElementsByClassName('usa-step-indicator__heading-text')[0].focus();
-  });
+  }, []);
 
   return (
     <div className="usa-step-indicator__header">
       {/* <style>{`:focus{ outline: solid 1px red}`}</style> */}
       <h2 className="usa-step-indicator__heading">
         {/* eslint-disable-next-line jsx-a11y/no-noninteractive-tabindex */}
-        <span tabIndex="0" className="usa-step-indicator__heading-text">{label}</span>
+        <span tabIndex="0" key={key} className="usa-step-indicator__heading-text">{label}</span>
       </h2>
     </div>
   );
 };
 
 NavigatorHeader.propTypes = {
   label: PropTypes.string.isRequired,
+  key: PropTypes.string.isRequired,
```

Comment:
Can also remove this line

Reply:
And thus, my react ignorance is revealed :smile:

Reply:
Keys are a special prop that tells gives the component an id, when that id changes react knows it needs to re-render the component. Usually they are used in lists so react can track which items need to be updated/removed when a list changes https://reactjs.org/docs/lists-and-keys.html#keys

Reply:
Cool, thanks for the explanation.